File: coding_practice(programmers.co.kr)/Level1/두 정수 사이의 합/두정수사이의합.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

long long solution(int a, int b)
{
    long long answer = 0;
    if (a > b)
    {
        int i = a - b;
        for (; i + 1 != 0; i--)
        {
            answer += b;
            b++;
        }
        return (answer);
    }
    for (int i = b - a; i + 1 != 0; i--)
        {
            answer += a;
            a++;
        }   
    return answer;
}

File: coding_practice(programmers.co.kr)/Level1/두 정수 사이의 합/두정수사이의합.py
def solution(a, b):
    answer = 0
    if b > a:
        i = b - a + 1
        while i != 0:
            answer += a
            a += 1
            i -= 1
    else:
        i = a - b + 1
        while i != 0:
            answer += b
            b += 1
            i -= 1
    return answer
